### PR TITLE
node pmaster added

### PR DESCRIPTION
**Pull request by oleksdiam**
node 'pmaster' added.
It allows gathering logs from puppet server

### CODE REVIEW

File: manifests/site.pp [hunk 1]
```diff
@@ -32,8 +34,8 @@
 node 'zabbix' {
   include zabbixsrv
 }
-node 'db'{
-  include zabbixagent
+node 'binddns'{
+  include role::binddns
```
Comment (turnopil) [inline manifests/site.pp:38]: cant find role::binddns in manifests directory. we really need a dns node for tomorrow demo?

**Review by helgard88 — APPROVED** [review]

File: manifests/site.pp (hunk 1 above)
Comment (oleksdiam) [reply manifests/site.pp:38]: No, we don't. But, anyway we need it in the future and it's already present in vagranfile.
And in this case i'll be able test dns module with no changes in site.pp.

Comment (oleksdiam) [reply manifests/site.pp:38]: I can add binddns role, profile and module right now, but I don't sure it's working properly for now.

Comment (turnopil) [reply manifests/site.pp:38]: It's better to do in different dev branch, example - dns :)

Comment (oleksdiam) [reply manifests/site.pp:38]: I got already a new local branch for dns, but it's not tested yet, so now we talking about site.pp only.

**Pull request closed without merging**